CONTRIBUTING.md

```diff
@@ -40,7 +40,7 @@
 1. Open a new issue explaining the feature.
 1. If the issue collects enough possitive feedback, fork the project and create a new feature branch from `master` to apply your changes.
 1. Once you are ready, open a new GitHub pull request to merge your changes back into `master`.
-1. Before submitting, please ensure that your changes follow the current coding conventions for this project.
+1. Before submitting, please ensure that your changes follow the current coding conventions for this project and run the test suite including your own tests to ensure all of them pass.
```

Comment:
I would say that the _including your own tests_ part is a bit redundant but I 100% agree about making explicit reference to run the test suite 👍 

What do you think about:

```markdown
1. Before submitting, please ensure that your changes follow the current coding
   conventions for this project and the [test suite](#running-tests) passes without errors.
```
Regarding to add new tests to the code base, I would reinforce this by adding explicit reference into the _creating a patch_ section (something like: remember to add test cases to ensure that the issue has been fixed) and _adding a new feature_ (something like: please, add or update any test case which may be affected by these changes).